storm-client/src/jvm/org/apache/storm/utils/ShellUtils.java

```diff
@@ -135,8 +135,10 @@ protected void setWorkingDirectory(File dir) {
 
     /** a Unix command to get the current user's groups list */
     public static String[] getGroupsCommand() {
-        return (WINDOWS)? new String[]{"cmd", "/c", "groups"}
-        : new String[]{"bash", "-c", "groups"};
```

Comment:
Is this a regression or did it really not work?

Reply:
Running "cmd /c groups" or just "groups" on my Windows 10 install just gives me a command not found error. I googled it a bit too, and didn't turn up any references to a "groups" command on Windows. I don't see any references to this method in the codebase either.